Tests/PathKitTests/PathKitSpec.swift

```diff
@@ -139,12 +139,20 @@ describe("PathKit") {
     try expect(path.normalize()) == Path("/usr/bin/swift")
   }
 
-#if !os(Linux)
   $0.it("can be abbreviated") {
+#if os(Linux)
```

Comment:
Since `HOME` will also be set on macOS. I think we can unify the test here so the same test covers both platforms. Would you be able to update the test?